src/components/wb-playground/kyn-modal-wb/kyn-modal-wb.ts

```diff
@@ -0,0 +1,188 @@
+import { LitElement, html, PropertyValues } from 'lit';
+import { customElement, state, query, property } from 'lit/decorators.js';
+import _formatDate from '../utils/formatDate';
+import ModalStyles from './kyn-modal-wb.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/link';
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+import '@kyndryl-design-system/shidoka-foundation/components/icon';
+
+import closeIcon from '@carbon/icons/es/close/20';
+
+/**
+ * Modal.
+ * @slot anchor - Slot for the anchor button content.
+ */
+@customElement('kyn-modal-wb')
+export class KynModalWb extends LitElement {
+  static override styles = [ModalStyles];
+
+  /** Date formatting function
+   * @internal
+   */
+  private static _formatDate = _formatDate;
+
+  /**
+   * Modal size `'auto'`, `'sm'`, `'md'`, or `'lg'`.
+   * @type {string}
+   * @default 'sm'
+   */
+  @property({ type: String })
+  size = 'sm';
+
+  /**
+   * Title/heading text, required.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  modalTitle = '';
+
+  /**
+   * Subheading text, optional.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  subheader = '';
+
+  /**
+   * Body text, optional.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  modalBody = '';
+
+  /**
+   * Modal open state.
+   * @type {boolean}
+   * @default false
+   */
+  @property({ type: Boolean })
+  showModal = false;
+
+  /**
+   * Subheader href to determine if is link.
+   * @type {string}
+   * @default ''
+   */
+  @property({ type: String })
+  subheaderHref = '';
+
+  /** Formatted timestamp value
+   * @internal
+   */
+  @state()
+  timestamp: string = KynModalWb._formatDate(Date.now());
```

Comment:
For displaying dates and time, I always recommend using one of the native locale formatting functions rather than something custom, ex: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString